csv-data-management/extract_data_from_csv.py

```diff
@@ -171,6 +171,47 @@ def concat_two_csv_and_extract_data(config, params):
 
     except Exception as Err:
         logger.error('Error in extract_data_from_csv(): %s' % Err)
+
+def join_two_csv_and_extract_data(config, params):
+    try:
+        fileOneIRI = handle_params(params,params.get('file_one_value'))
+        fileOnePath = join('/tmp', download_file_from_cyops(fileOneIRI)['cyops_file_path'])
+        fileTwoIRI = handle_params(params,params.get('file_two_value'))
+        fileTwoPath = join('/tmp', download_file_from_cyops(fileTwoIRI)['cyops_file_path'])
+
+
+        df1 = _read_and_return_ds(fileOnePath,params,config)
+        df2=  _read_and_return_ds(fileTwoPath,params,config)
+
+        #Merge both files
+        combined_recordSet =df1.join(df2,lsuffix='_FirstFile', rsuffix='_SecondFile')    
+
+        # If user has selected to deduplicate recordset
+        try:
+            if params.get('deDupValuesOn'):
+                deDupValuesOn = params.get('deDupValuesOn')
+                deDupValuesOn = deDupValuesOn.split(",")
+                combined_recordSet=combined_recordSet.drop_duplicates(subset=deDupValuesOn, keep='first')
+        except Exception as Err:
+            logger.error('Error in deduplicating data  extract_data_from_csv(): %s' % Err)
```

Comment:
After exception, why do you want code to be continue from here?

Reply:
addressed PR